siac/gatewaycmd.go

```diff
@@ -14,22 +14,36 @@ var (
 	gatewayCmd = &cobra.Command{
 		Use:   "gateway",
 		Short: "Perform gateway actions",
-		Long:  "Add or remove a peer, view the current peer list, or synchronize to the network.",
+		Long:  "View and manage the gateway's connected peers.",
 		Run:   wrap(gatewaycmd),
 	}
 
-	gatewayAddCmd = &cobra.Command{
-		Use:   "add [address]",
-		Short: "Add a peer",
-		Long:  "Add a new peer to the peer list.",
-		Run:   wrap(gatewayaddcmd),
+	gatewayConnectCmd = &cobra.Command{
+		Use:   "connect [address]",
+		Short: "Connect to a peer",
+		Long:  "Connect to a peer and add it to the node list.",
+		Run:   wrap(gatewayconnectcmd),
 	}
 
-	gatewayRemoveCmd = &cobra.Command{
-		Use:   "remove [address]",
-		Short: "Remove a peer",
-		Long:  "Remove a peer from the peer list.",
-		Run:   wrap(gatewayremovecmd),
+	// COMPATv0.6.0
+	gatewayDeprecatedAddCmd = &cobra.Command{
+		Use:        "add [address]",
+		Deprecated: "use `siac gateway connect` instead.",
+		Run:        wrap(gatewayconnectcmd),
+	}
+
+	gatewayDisconnectCmd = &cobra.Command{
+		Use:   "disconnect [address]",
+		Short: "Disconnect from a peer",
+		Long:  "Disconnect from a peer. Does not remove the peer from the node list.",
+		Run:   wrap(gatewaydisconnectcmd),
+	}
+
+	// COMPATv0.6.0
+	gatewayDeprecatedRemoveCmd = &cobra.Command{
+		Use:        "remove [address]",
+		Deprecated: "use `siac gateway disconnect` instead.",
+		Run:        wrap(gatewaydisconnectcmd),
```

Comment:
If we want to remove the `remove` command eventually we should deprecate it instead of aliasing it so that the user is warned that its deprecated when they try to use it.